src/Microsoft.Identity.Web.Certificateless/ClientAssertionDescription.cs

```diff
@@ -11,22 +11,12 @@ namespace Microsoft.Identity.Web
     /// Description of a client assertion in the application configuration.
     /// See https://aka.ms/ms-id-web/client-assertions.
     /// </summary>
-    internal class ClientAssertionDescription
+    public class ClientAssertionDescription
```

Comment:
I have not renamed it. Maybe we want to rename it to BaseClientAssertion?

Reply:
makes sense